rules/windows/defense_evasion_masquerading_as_elastic_endpoint_process.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/08/24"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+A suspicious Elastic Endpoint Parent Process was detected. Verify process details such as hash, commandline, network
+connections, file writes, parent and child process details as well.
```

Comment:
```suggestion
A suspicious Elastic endpoint agent parent process was detected. Verify process details such as hash, command line, network
connections, file writes, parent and child process details as well.
```

Reply:
@Samirbous, you mentioned that this was to target in-memory or process-hollowing attempts (as opposed to using illegitimate executables, in which case we would just check signer information). Can you add mention of this within the description to clarify